packages/components/src/font-size-picker/utils.js

```diff
@@ -15,13 +15,11 @@ const CUSTOM_FONT_SIZE_OPTION = {
 };
 
 /**
- * In case we have at most five font sizes, where at least one the them
- * contain a complex css value(clamp, var, etc..) show a incremental sequence
- * of numbers as a label of the font size. We do this because complex css values
- * cannot be caluclated properly and the incremental sequence of numbers as labels
- * can help the user better mentally map the different available font sizes.
+ * In case we have at most five font sizes, show a `T-shirt size`
+ * alias as a label of the font size. The label assumes that the font sizes
+ * are ordered accordingly - from smallest to largest.
  */
-const FONT_SIZES_ALIASES = [ '1', '2', '3', '4', '5' ];
+const FONT_SIZES_ALIASES = [ 'S', 'M', 'L', 'XL', 'XXL' ];
```

Comment:
Also these label  probably need to be translatable.

Reply:
Good call! Will do